docs/reach/rs2/p4p-rtk.md

```diff
@@ -0,0 +1,76 @@
+
+## Registration in RTK2Go
+
+RTK2Go is a free-to-use NTRIP caster provider which enables you to make your own Mount Point under its domain. The service requires a free registration procedure:
+
+- Navigate to the RTK2Go site and choose *"Please register before use"* button:
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/rtk2go-start.png" style="width: 600px;"/></p>
+
+- Fill in your registration details:
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/rtk2go-registration.png" style="width: 600px;"/></p>
+
+You will receive a confirmation email with the name of your Mount Point and passwords for base and rovers. RTK2Go allows multiple rovers to be connected to one base station, that is why you can make use of it for many devices. For instance, you can send correction data to other Reach rovers to collect accurate data in the field including placing GCPs.
+
+## Configuring Reach RS2
+
+Reach RS2 acts as a base, sending the corrections via NTRIP service. 
+
+- Place the Reach RS2 onto the tripod and provide it with the clear sky view
+- Provide the receiver with an Internet access according to [this guide](../connecting-to-the-internet/)
+- Navigate to the Base Mode tab and insert the NTRIP credentials for the base station
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/rv-base-mode.png" style="width: 800px;"/></p>
+
+## Configuring DJI Phantom 4 RTK
+
+- Open GS RTK
+- Select Plan or Fly menu. If you choose Plan, select a planning method
+- Open Settings by tapping on 3 dots button at the top right corner
+- Go to RTK Settings tab
+- In RTK Service Type, choose Custom network RTK and fill in NTRIP credentials form
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/dji-correction-input.png" style="width: 600px;"/></p>
+
+- Tap Connect button. The "Network RTK server connection successful" will appear:
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/dji-correction-success.png" style="width: 600px;"/></p>
+
+- Base is successfully connected
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/dji-sattelites.png" style="width: 600px;"/></p>
+
+Phantom shows the base's satellites number only if the RTK solution status is fix.
+
+## Performing the Flight
+
+Once the RS2 and drone are set, you can perform your flight. Provide the Phantom 4 with a clear sky view and proceed to the mission:
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/dji-flight.png" style="width: 600px;"/></p>
+
+You can check the progress of your flight using the map on the bottom left corner of the UI:
+
+<p style="text-align:center"><img src="../img/reachrs2/p4p/dji-plan.png" style="width: 600px;"/></p>
+
+Once the flight is completed, you can download the precisely geotagged photos and use them to create your orthophotos.
+
+
+Further reading:
+
+- [Connecting Reach RS2 to Internet](../connecting-to-the-internet/)
```

Comment:
- [Connecting Reach RS2 to Internet](../connecting-to-the-internet/) - remove
- [Base mode](../common/reachview/base-mode/) - remove
- [Working with NTRIP service](../ntrip-workflow/) - remove

We need to have the following articles here:
* How RTK works
* Placing the base
* Placing GCPs in RTK